.github/workflows/integration-tests.yml

```diff
@@ -61,7 +61,6 @@ jobs:
         os:
           - "windows-2022"
         dotnet-version:
-          - "net472"
```

Comment:
and 12 people cried out in horror and disgust

Reply:
As well they should